google/cloud/bigtable/internal/readrowsparser.cc

```diff
@@ -36,12 +36,12 @@ void ReadRowsParser::HandleChunk(ReadRowsResponse_CellChunk chunk,
   }
 
   if (!chunk.row_key().empty()) {
-    if (last_seen_row_key_.compare(chunk.row_key()) >= 0) {
+    if (CompareRowKey(last_seen_row_key_, chunk.row_key()) >= 0) {
       status = grpc::Status(grpc::StatusCode::INTERNAL,
                             "Row keys are expected in increasing order");
       return;
     }
-    chunk.mutable_row_key()->swap(cell_.row);
+    std::swap(*chunk.mutable_row_key(), cell_.row);
```

Comment:
We probably want to use the "swap idiom" here, so that the most efficient implementation of swap is chosen. This may actually be meaningful here since cell_.row may be the google-internal or external type. Something like:

```cc
using std::swap;
swap(*chunk.mutable_row_key(), cell_.row);
```

Reply:
Done.